spec/models/provider_enrichment_spec.rb

```diff
@@ -22,14 +22,45 @@
   end
 
   describe 'auditing' do
-    it { should be_audited }
+    it { should be_audited.except(:accrediting_provider_enrichments) }
 
     it 'can link to Provider' do
       provider = create(:provider)
       provider_enrichment = create(:provider_enrichment, provider: provider)
 
       expect(provider_enrichment.audits.last.associated).to eq(provider)
     end
+
+    context 'when dealing with `accrediting_provider_enrichments`' do
+      let(:accrediting_provider_enrichment) do
+        {
+          "errors" => {}, "Description" => "A test value", "validation_context" => nil
```

Comment:
```suggestion
          "UcasProviderCode" => "ABC", "Description" => "A test value"
```
That is more aligned to real life


wondering `AccreditingProviderEnrichment::ArraySerializer.load().dump()`